ngmix/guessers.py

```
@@ -803,6 +803,10 @@ def _get_T_flux_from_moms(self, obs):
         wt = GMixModel([0.0, 0.0, 0.0, 0.0, Tweight, 1.0], "gauss")
 
         res = wt.get_weighted_moments(obs=obs, maxrad=1.0e9)
+        assert res['flags'] == 0, 'weighted moments failed'
```

This should raise an appropriate error.